lib/runtime/git_hooks.dart

```diff
@@ -11,15 +11,19 @@ class GitHooks {
 
   /// create files from dart codes.
   /// [targetPath] is the absolute path
-  static void init({String targetPath}) async {
+  static void init({String? targetPath}) async {
     await Process.run('git_hooks', ['-v']).catchError((onError) async {
-      var result = await Process.run('pub', [
+      final args = [
         'global',
         'activate',
         '--source',
         'path',
-        Utils.getOwnPath()
-      ]).catchError((onError) {
+      ];
+      final ownPath = Utils.getOwnPath();
+      if (ownPath is String) {
+        args.add(ownPath);
+      }
+      var result = await Process.run('pub', args).catchError((onError) {
```

Comment:
Accounting for when `Utils.getOwnPath()` returns `null`. Let me know if this is correct.